invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/modal.overrides

```diff
@@ -115,19 +115,22 @@
     border-top: @actionBorder;
     text-align: @actionAlign;
   }
-  .content {
-  display: block;
-  width: 100%;
-  font-size: @contentFontSize;
-  line-height: @contentLineHeight;
-  padding: @contentPadding;
-  background: @contentBackground;
-}
+
+  & > .content {
```

Comment:
this was done on purpose, it will break other modals, check the comment in line 110.
by doing this change you make it identical with original semantic ui code

Reply:
Do you know what it was fixing? Because it's causing a lot of inheritance issues in the modals, since `.content`is also a sub class of other classes.